docs/network-setup.md

```diff
@@ -51,14 +51,20 @@ sudo iptables-save > iptables.rules.old
 Before starting the guest, configure the network interface using Firecracker's
 API:
 
+**Note** If you use rootfs from the [getting started guide](getting-started.md)
```

Comment:
```diff
-**Note** If you use rootfs from the [getting started guide](getting-started.md)
-you need to use specific `MAC` address like `06:00:AC:10:00:02`. In this `MAC`
-the last 4 bytes (`AC:10:00:02`) will represent the ip address of the guest. In default 
-case it is `172.16.0.2`. Otherwise you can skip `guest_mac` field for network 
-configuration. This way guest will generate random mac on startup.
+**Note:** If you use the rootfs from the [getting started guide](getting-started.md),
+you need to use a specific `MAC` address like `06:00:AC:10:00:02`. In this `MAC`
+address, the last 4 bytes (`AC:10:00:02`) will represent the IP address of the
+guest. In the default case, it is `172.16.0.2`. Otherwise, you can skip the
+`guest_mac` field for network configuration. This way, the guest will generate a
+random MAC address on startup.
```

Reply:
done